droid-binary/pom.xml

```diff
@@ -4,7 +4,7 @@
     <parent>
         <artifactId>droid-parent</artifactId>
         <groupId>uk.gov.nationalarchives</groupId>
-        <version>6.5-SNAPSHOT</version>
+        <version>6.5-VITAM</version>
```

Comment:
Can you remove these version changes please?

Reply:
Yes for sure...